code/index.html

```diff
@@ -1,22 +1,110 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Skaftö Open</title>
+  <link rel="stylesheet" href="./style.css" />
+  <!-- dont forget the meta tag for responsiveness -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+</head>
+
+<header>
+  <!-- Some kind of logo and navbar -->
+  <img class="banner" src="Skaftoopenmagazine.JPG" alt="Skaftö Open Magazine">
+
+  <nav>
+    <a class="links">
+      <a
+        href="https://www.google.com/search?q=skaft%C3%B6+open&oq=skaft%C3%B6+open+&aqs=chrome.0.69i59l2j35i39j0l3j69i61l2.4334j0j7&sourceid=chrome&ie=UTF-8">News</a>
+      </div>
```

Comment:
I'm not completely sure where the opening div-tag to this tag is. There seem to be a few, perhaps you changed from `<div>`-tags to `<a>`-tags and forgot to remove them? 🤔